src/Symfony/Component/Serializer/Encoder/XmlEncoder.php

```diff
@@ -52,6 +52,7 @@ class XmlEncoder implements EncoderInterface, DecoderInterface, NormalizationAwa
     public const STANDALONE = 'xml_standalone';
     public const TYPE_CAST_ATTRIBUTES = 'xml_type_cast_attributes';
     public const VERSION = 'xml_version';
+    public const NULLIFY_EMPTY_TAGS = 'nullify_empty_tags';
```

Comment:
Could update the XmlEncoderContextBuilder as well?

Reply:
done